ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/PlatformCvesByStatusSummaryCard.tsx

```diff
@@ -23,6 +23,11 @@ const statusDisplays = [
     },
 ] as const;
 
+const statusHiddenText = {
+    Fixable: 'Fixable hidden',
+    'Not fixable': 'Not fixable hidden',
```

Comment:
Good enough. In case you need a smile, somehow my first impression was contrast:
* Fixable are hidden
* Fixable are not hidden

Maybe because I recently started refreshing my reading ability in a language that has flexible word order.

Reply:
Yeah, I was thinking something like "Unfixable CVEs hidden", but that would be a terminology mismatch with the "Not fixable" text we are using everywhere else. :/

An improvement but maybe not 100% of the way to what we want.